src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java

```diff
@@ -90,5 +90,42 @@ void testGetListingByIdWhenListingDoesNotExistThenReturnNotFound() throws Except
     }
 
 
+    // Delete Testings
+    @Test
+    void testDeleteListing_Success() throws Exception {
+        Long listingID = 1L;
+
+        mockMvc.perform(MockMvcRequestBuilders.delete("/api/v1/listings/{listingID}", listingID))
+                .andExpect(MockMvcResultMatchers.status().isNoContent());
+
+        verify(mockListingService).deleteListingById(listingID);
+    }
```

Comment:
use times from Mockito to see that it is being called only once.
Change the expected code to accepted( we accepted the request to delete and was successful)